lib/coinbase/address/external_address.rb

```diff
@@ -69,20 +65,20 @@ def staking_balances(asset_id, mode: :default, options: {})
           }
         )
       end.context
-      if context_model.stakeable_balance.empty? || context_model.stakeable_balance.nil?
-        context_model.stakeable_balance = '0'
-      end
-      if context_model.unstakeable_balance.empty? || context_model.unstakeable_balance.nil?
-        context_model.unstakeable_balance = '0'
-      end
-      if context_model.claimable_balance.empty? || context_model.claimable_balance.nil?
-        context_model.claimable_balance = '0'
-      end
 
       {
-        stakeable_balance: asset.from_atomic_amount(context_model.stakeable_balance.to_i),
-        unstakeable_balance: asset.from_atomic_amount(context_model.unstakeable_balance.to_i),
-        claimable_balance: asset.from_atomic_amount(context_model.claimable_balance.to_i)
+        stakeable_balance: Coinbase::Balance.from_model_and_asset_id(
+          context_model.stakeable_balance,
```

Comment:
Just thinking & seeing this now, right now this object can be nil. It probably shouldn't be.